src/services/utils.js

```diff
@@ -78,18 +78,6 @@ export const getRandomInt = (min, max) => Math.floor(Math.random() * (max - min
 
 export const circleToPath = (cx, cy, r) => `M ${cx} ${cy} m -${r}, 0 a ${r},${r} 0 1,0 ${r * 2},0 a ${r},${r} 0 1,0 -${r * 2},0`;
 
-export const loadJSAsync = (e, n, o) => {
```

Comment:
This is a breaking change possibly. Not sure anyone else is using it atm though.

Reply:
I don't think anyone is, but I still think we should follow the SemVer when we release this and bump the major version.